### ISSUE TITLE
Resnet50 pretrained model required

### ISSUE THREAD

**Issue by yahuuuuuuu** [zhangchbin/RCIL#14]
While replicating your code, I was wondering if I could use resnet50 (which you provided in parser) or renset18, but they need a copy of the pre-trained weights, so where can I get them?

**Comment by zhangchbin** [comment]
We only used the [ResNet-101_iabn](https://github.com/arthurdouillard/CVPR2021_PLOP/releases/download/v1.0/resnet101_iabn_sync.pth.tar) that uses non-standard batch normalization to save training memory. So if you try to use Resnet-50 or Resnet-18 with standard batch normalization, please modify the merge operation between two branches. Thanks.

**Comment by zhangchbin** [comment]
hi, @yahuuuuuuu 
We found that high version pytorch can not reach the supposed performance due to the bug caused by InplaceABNSync, which has been fixed in our latest commitment. You can refer to our latest news. Sorry for the inconvenience caused.
